back-end/app/Models/User.php

```diff
@@ -45,11 +30,10 @@ class User extends Authenticatable
     protected function casts(): array
     {
         return [
-            'email_verified_at' => 'datetime',
-            'password' => 'hashed',
+            'UserPassword' => 'string',
```

Comment:
password inplaats van userPassword

```suggestion
            'password' => 'string',
```